### PR TITLE
Move reconcile testing out of aftersuite

### PR DESCRIPTION
**Pull request by clyang82**
- Move reconcile testing out of aftersuite
- Disable switch from Basic to High mode

https://github.com/open-cluster-management/backlog/issues/10620

Signed-off-by: clyang82 <chuyang@redhat.com>

### CODE REVIEW

File: pkg/utils/mco_deploy.go [hunk 1]
```diff
@@ -418,7 +418,6 @@ func ModifyMCOCR(opt TestOptions) error {
 	spec := mco.Object["spec"].(map[string]interface{})
 	spec["retentionResolutionRaw"] = "3d"
 	spec["nodeSelector"] = map[string]string{"kubernetes.io/os": "linux"}
-	spec["availabilityConfig"] = "Basic"
```
Comment (songleo) [inline pkg/utils/mco_deploy.go:421]: I do not understand why we delete this line?

Comment (clyang82) [reply pkg/utils/mco_deploy.go:421]: I found that this case can be failure in most case

Comment (songleo) [reply pkg/utils/mco_deploy.go:421]: you mean we do not need this case? maybe we can add retry func to handle it.

Comment (clyang82) [reply pkg/utils/mco_deploy.go:421]: that is not a P1 case for release-2.2. You may know that it spends more time to switch from Basic to High and back. I think we can just disable it to reduce the failure.

**Review by songleo — APPROVED** [review]

**Pull request merged**